GoedeDoelenHelpen/Startup.cs

```diff
@@ -127,6 +128,17 @@ public void ConfigureServices(IServiceCollection services)
             {
                 options.FileProviders.Add(fileProvider);
             });
+
+            //Facebook koppeling
+            
+
+            services.AddAuthentication().AddFacebook(facebookOptions =>
```

Comment:
deed je niet alles is de frontend? is dit nog nodig?

Reply:
Je hebt gelijk, ik heb het verwijderd!